src/web/components/SharingPermission/SharingPermissionsTable.tsx

```diff
@@ -81,6 +101,7 @@ export function SharingPermissionsTable({
     }
   }, [checkedParticipants.size, isSelectedAll]);
 
+  if (isLoading) return <Loading />;
   const deletePermissionBtn = (
```

Comment:
Should we also include some conditional copy in this `dialog-body-section` to inform the user that if the participant has been shared by type, then it will continue to be shared with despite deletion?

Reply:
Yes, I remember we have that conversation before, but I couldn't find the copy for that anymore. I will come up with something and let's leave it for Gen to review the copy